mojaloop-simulator/templates/ingress.yaml

```diff
@@ -21,15 +21,15 @@ spec:
       - host: {{ $host }}
         http:
           paths:
-          - path: /sim/{{ $name }}/outbound{{ if $.Values.ingress.modernIngressController }}{{ $.Values.ingress.modernIngressControllerRegex }}{{ end }}
+          - path: /sim/{{ $name }}/outbound{{ $.Values.ingress.ingressPathRewriteRegex }}
             backend:
               serviceName: sim-{{ $name }}-scheme-adapter
               servicePort: outboundapi
-          - path: /sim/{{ $name }}/inbound{{ if $.Values.ingress.modernIngressController }}{{ $.Values.ingress.modernIngressControllerRegex }}{{ end }}
+          - path: /sim/{{ $name }}/inbound{{ $.Values.ingress.ingressPathRewriteRegex }}
             backend:
               serviceName: sim-{{ $name }}-scheme-adapter
               servicePort: inboundapi
-          - path: /sim/{{ $name }}/test{{ if $.Values.ingress.modernIngressController }}{{ $.Values.ingress.modernIngressControllerRegex }}{{ end }}
+          - path: /sim/{{ $name }}/test{{ $.Values.ingress.ingressPathRewriteRegex }}
```

Comment:
but for ex. /sim/{{ $name }}/test/(/|$)(.*) does not work for older ingress controller versions right

Reply:
Yup. That's addressed by the comment here: https://github.com/ksatya77/helm/pull/2/files#diff-a272c8fdad745d1de5b9769bfd8fe4acR117-R122